doc/src/manual/packages.md

```diff
@@ -171,6 +171,14 @@ git config --global url."https://".insteadOf git://
 
 However, this change will be system-wide and thus the use of [`Pkg.setprotocol!()`](@ref) is preferable.
 
+!!! note
+    The package manager functions accept package names with the `.jl` extension:
```

Comment:
I'd probably word this to say that it's possible but is stripped out internally so as not to encourage this style of use. Maybe "The package manager functions also accept the `.jl` suffix on package names, though the suffix is stripped out internally."